pkg/cli/prompt/list/list.go

```diff
@@ -134,6 +131,9 @@ func (m ListModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 	switch msg := msg.(type) {
 	case tea.WindowSizeMsg:
 		m.width = msg.Width
+		// setting the width and height of the list model as terminal dimensions changes.
+		// setting the height to 25% of the height of the terminal height.
```

Comment:
why 25% specifically?

Reply:
we want to change the dimension of the list model as terminal dimensions changes, and I am setting it to 25% because i figured its close to the existing size of the list model on a full terminal window.